quickstart.Dockerfile

```
@@ -1,44 +1,44 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
 
 # Environment variable
-ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/usr/share/elasticsearch/users.yml
-ENV TEAM_PASSWORD=1234
-ENV ARGILLA_PASSWORD=1234
-ENV TEAM_API_KEY=team.apikey
-ENV ARGILLA_API_KEY=argilla.apikey
+ENV ADMIN_USERNAME=admin
+ENV ADMIN_PASSWORD=11223344
+ENV ADMIN_API_KEY=team.apikey
+
+ENV ANNOTATOR_USERNAME=argilla
+ENV ANNOTATOR_PASSWORD=11223344
+
+ENV ARGILLA_WORKSPACE=$ANNOTATOR_USERNAME
 ENV LOAD_DATASETS=full
 ENV UVICORN_PORT=6900
+
 ENV xpack.security.enabled=false
 ENV cluster.routing.allocation.disk.threshold_enabled=false
 ENV discovery.type=single-node
 ENV ES_JAVA_OPTS=-'Xms512m -Xmx512m'
 
 USER root
 
+COPY scripts/start_quickstart_argilla.sh /
+COPY scripts/load_data.py /
+COPY dist/*.whl /packages/
+
+
 # Install packages
 RUN apt update && \
     apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl && \
     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
     python3.9 get-pip.py && \
-    pip3 install datasets
-
-COPY scripts/start_quickstart_argilla.sh /
-COPY scripts/load_data.py /
-COPY dist/*.whl /packages/
-
-# Install argilla
-RUN chmod +x /start_quickstart_argilla.sh && \
+    pip3 install datasets \
+    # Install Argilla
+    && chmod +x /start_quickstart_argilla.sh && \
```

If I'm not wrong this is allowed right? (putting comments between lines and don't alter anything):

```suggestion
    pip3 install datasets && \
    # Install Argilla
    chmod +x /start_quickstart_argilla.sh && \
```